tests/model/SiteTreeTest.php

```diff
@@ -858,6 +858,25 @@ public function testAllowedChildren() {
 		);
 	}
 
+	public function testAllowedParents() {
+		$page = new Page();
+		$parent1 = new SiteTreeTest_SpecificParent1();
+		$parent2 = new SiteTreeTest_SpecificParent2();
+
+		// Ensure that none of the "Specific" parents and children show up as allowed children on a generic Page object.
+		$pageChildren = $page->allowedChildren();
+		$this->assertEquals(false, in_array("SiteTreeTest_SpecificParent1", $pageChildren));
```

Comment:
Ok, I'll use `assertNotContains()` since I'm trying to ensure it's not in the list.